### PR TITLE
move pyfranca_cpp into a separate project

### PR DESCRIPTION
**Pull request by Rosi2143**
created new repository https://github.com/Rosi2143/pyfranca_cpp and re-use this repository as submodule here.

P.S. @gunnarx: I'm fine to transfer the repository https://github.com/Rosi2143/pyfranca_cpp to you again.

Signed-off-by: rmi2hi <michael.rossner@de.bosch.com>

### CODE REVIEW

**Comment by gunnarx** [comment]
@Rosi2143 
This is fine.  Happy to let you take over the maintainership of this code since I do not work with it a lot.  

I'd like to recommend that you track this project however:  https://github.com/COVESA/vehicle_service_catalog  .   I don't know exactly what kind of work you do and if the creation of a common service catalog for vehicles is interesting (butr if you are in automotive, you should join the phone-meeting discussions!)....  but even independent of the actual catalog, 
the project is working on unify the usage of Interface Description Languages.

In other words you may find one or several of these things:
- A YAML-based IDL that you might like (or not).  It is very aligned with Franca, and similar to Franca in features, but it adds some new features also
- Interoperability / translation to and from Franca is planned (need resources)
- New code-generators are likely to be developed there instead, so your input is welcome if there is a fit.
